apps/omg_watcher/lib/watcher.ex

```diff
@@ -13,5 +13,9 @@
 # limitations under the License.
 
 defmodule OMG.Watcher do
-  @moduledoc false
+  @moduledoc """
+  Watcher is responsible for syncing and validating the child chain, and providing a secure API to it.
```

Comment:
`API to do it`

Reply:
`to it` please read the sentence again. The watcher is responsible for providing a secure API to the child chain. But I'll rephrase to `interface` instead, to be more precise and perhapse less reado prone